inst/rmd/h/default.html

```diff
@@ -285,27 +300,30 @@
   var menuAnchor = $$('a[href="' + href + '"]');
 
   // mark it active
-  menuAnchor.parent().addClass('active');
+  menuAnchor.tab('show');
 
   // if it's got a parent navbar menu mark it active as well
   menuAnchor.closest('li.dropdown').addClass('active');
```

Comment:
This line is probably not needed, I think `menuAnchor` takes care of it.

Reply:
It's currently needed when `bootstrap_version: 4`. That's because the navbar dropdown markup we generate in R is not exactly what BS4 wants/expects (namely it seems BS4 wants `a.dropdown-item` instead of `li a.dropdown-item` for the active class to be added properly to the `dropdown`), but even if we did generate the 'correct' markup, it seems to still won't work with `.dropdown-submenu`.